config/prod/bwhite-beat-aml-bioc-new.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
```

Comment:
Remove this file

Reply:
The request is in bwhite-beat-aml-bioc-again.yaml (which uses template_path: remote/managed-ec2-linux-v1.j2 and should be correct) and not bwhite-beat-aml-bioc-new.yaml (which uses the incorrect template_path: remote/managed-ec2-ubuntu-v3.j2). Is bwhite-beat-aml-bioc-again.yaml ready to go? Do I need to remove bwhite-beat-aml-bioc-new.yaml?

Reply:
Yes you need to remove the bwhite-beat-aml...new.yaml.